src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -371,6 +381,40 @@ def attestationChallenge(self) -> bytes:
 
         return bytes(buf)
 
+    @property
+    def sessionAllowsLargePayload(self) -> bool:
```

Comment:
Properties are typically capitalized:

```suggestion
    def SessionAllowsLargePayload(self) -> bool:
```

Reply:
Hm, I see all properties are lower case here :see_no_evil: 

I think we should fix this, but should be a follow up PR. For this PR, you can leave as is (and the follow up will change), or already adopt the capitalize stance, either way works for me.

Reply:
Yes, all existing ones are lowercase and that is the convention I followed. I will leave this one as-is for now.